esx_service/vmdk_ops.py

```diff
@@ -475,24 +475,33 @@ def get_vol_path(datastore, tenant_name=None):
     if os.path.isdir(path):
         # If the path exists then return it as is.
         logging.debug("Found %s, returning", path)
-        return path
+        return path, None
 
     if not os.path.isdir(dock_vol_path):
         # The osfs tools are usable for DOCK_VOLS_DIR on all datastores
         cmd = "{0} {1}".format(OSFS_MKDIR_CMD, dock_vol_path)
         rc, out = RunCommand(cmd)
-        if rc != 0:
-            logging.warning("Failed to create %s", dock_vol_path)
-            return None
+        errMsg = None
+        if rc == 13:
+            errMsg = "{0} is not writable and hence ./dockvol creation failed".format(datastore)
+        elif rc == 30:
+            errMsg = "Permission denied to create ./dockvol on {}".format(datastore)
+        else:
+            logging.warning("Failed to create %s", path)
+            errMsg = "Failed to initialize volume path {0}".format(path)
```

Comment:
different logging and errmsg strings looks strange. Can we use the same string ?